src/components/notebookRenderStrategy.tsx

```diff
@@ -12,9 +12,13 @@ import { NotebookOpenedIconSvg } from './icons/notebookOpenedIcon.svg';
 import { NotebookClosedIconSvg } from './icons/notebookClosedIcon.svg';
 import { ChevronSvg } from './icons/chevron.svg';
 import { CreateNewSectionNode } from './createNewSection/createNewSectionNode';
+import * as OneNoteApi from 'onenoteapi';
+import { SpinnerIconSvg } from './icons/spinnerIcon.svg';
+import { Strings } from '../strings';
 
 export class NotebookRenderStrategy implements ExpandableNodeRenderStrategy {
 	onClickBinded = () => {};
+	onExpandBinded = this.onExpand.bind(this);
```

Comment:
Don't use .bind - instead, use an arrow function. You can get rid of onExpandBinded and onClickBinded and change them to this syntax:

onExpand = () => { ... }
onClick = () => { ... }

On second thought... let's just leave this as is, I don't know how good the compilation in this one is so this suggestion might break IE, but if you write this in our newer codebases in VSTS, definitely use this approach.